src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random rand = new Random();
+    private ColorSupplier col = new ColorSupplier();
+
+    public Random getRand() {
+        return rand;
+    }
+
+    public void setRand(Random rand) {
+        this.rand = rand;
+    }
+
+    public ColorSupplier getCol() {
+        return col;
+    }
+
+    public void setCol(ColorSupplier col) {
+        this.col = col;
+    }
+
+    public Square squareIni() {

```

Comment:
```suggestion
    public Square getRandomSquare{

```

and so on. Do we need this methods to be public?